libaesnd/aesndlib.c

```diff
@@ -545,8 +545,8 @@ AESNDPB* AESND_AllocateVoiceWithArg(AESNDVoiceCallbackArg cb,void *cbArg)
 
 	_CPU_ISR_Disable(level);
 	for(i=0;i<MAX_VOICES;i++) {
-		pb = &__aesndvoicepb[i];
```

Comment:
i know its kinda out of scope of what you intended to do, but could you change the if structure here?
make it so it looks like this
```
for(i=0;i<MAX_VOICES;i++) {
  if(__aesndvoicepb[i].flags&VOICE_USED)
    continue;
  
  pb = &__aesndvoicepb[i];
  ...
```

Reply:
done